config/environments/development.rb

```diff
@@ -25,8 +25,8 @@
   }
   config.action_mailer.asset_host = IdentityConfig.store.mailer_domain_name
   config.action_mailer.raise_delivery_errors = false
-  config.action_mailer.smtp_settings = { address: ENV['SMTP_HOST'] || 'localhost', port: 1025 }
   config.action_mailer.show_previews = IdentityConfig.store.rails_mailer_previews_enabled
+  config.action_mailer.delivery_method = IdentityConfig.store.development_mailer_deliver_method
```

Comment:
The `letter_opener` docs include `perform_deliveries = true` in the setup instructions. Do we need it?

https://github.com/ryanb/letter_opener#rails-setup

Reply:
I'm not sure, but it probably can't hurt